NWNXLib/API/API/CTlkTable.hpp

```diff
@@ -47,7 +47,7 @@ struct CTlkTable
     virtual void GetTokenValue(const CTlkTableToken & cTlkTableToken, CExoString & sToken);
     BOOL FetchInternal(uint32_t strId, STR_RES & strRes, BOOL bParse);
     void CloseFile(CTlkFile * pTlkFile);
-    BOOL OpenFile(const CExoString & sFileName, CTlkFile * pTlkFile);
+    BOOL OpenFile(const CExoString & sFileName, CTlkFile ** pTlkFile);
```

Comment:
Technically, it's `CTlkFile *pTlkFile[2]`, and the CloseFile() should be updated as well.